examples/build.py

```diff
@@ -21,24 +22,31 @@
 def listify(a):
     return np.asarray(a, dtype='str').tolist()
 
-with open('new-emperor.html', 'w') as f, open('template.html') as temp:
-    template = Template(temp.read())
+# From http://stackoverflow.com/a/6196098
+pages = ['../emperor/support_files/templates/main-template.html',
+         'template.html']
+env = Environment()
+templates = dict((name, open(name, 'rb').read()) for name in pages)
```

Comment:
Oddly enough leaving these file handles open reflects in the files changing i.e. every time I re-run build.py I see a message on my editor about "This file has changed since you first loaded it ... etc" would you mind explicitly closing the file-handles?